en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/production-deployment-guideline.md

```diff
@@ -0,0 +1,288 @@
+```
+
+```yaml tab='Enforcer Truststore'
+wso2:
+  deployment:
+    gatewayRuntime:
+      enforcer:
+        security:
+          truststore:
+            - secretName: "controlplane-cert"
+              subPath: "wso2carbon.pem"
+            - secretName: "abc-certs"
+              subPath: "tls.crt"
+```
+
+## Change Default Passwords
+
+You can change the default passwords or environment variables by setting/overriding them by defining them in the field `envOverride`.
+These passwords or environment variables can be set directly as a plain text or can be referenced from a Secret or ConfigMap.
+This configuration is the same way you can define environment variables in Kubernetes Pods.
+
+In the following sample, the `enforcer_admin_pwd` is set using the `value` field and the `tm_admin_pwd` is referred using a secret. 
+
+```yaml
+wso2:
+  deployment:
+    gatewayRuntime:
+      enforcer:
+        envOverride:
+          - name: enforcer_admin_pwd
+            value: admin
+          - name: tm_admin_pwd
+            valueFrom:
+              secretKeyRef:
+                name: my-secret
+                key: tm_password
+```
+
+Following are the default values, update to read them from a Kubernetes secret.
+
+```yaml tab='Adapter'
+wso2:
+  deployment:
+    adapter:
+      envOverride:
+        - name: cp_admin_pwd
+          value: admin
+        - name: adapter_admin_pwd
+          value: admin
+```
+
+```yaml tab='Enforcer'
+wso2:
+  deployment:
+    gatewayRuntime:
+      enforcer:
+        envOverride:
+          - name: apim_admin_pwd
+            value: "admin"
+          - name: enforcer_admin_pwd
+            value: "admin"
+          - name: tm_admin_pwd
+            value: "admin"
+```
+
+## Configure Ingress Secrets
+
+By default, the helm chart installation of Choreo Connect creates default certificates for ingress resources.
+Providing TLS Secret name in the following values will replace those default certs with provided certs.
+
+!!! Note
+    Following certs should be [TLS secrets](https://kubernetes.io/docs/concepts/configuration/secret/#tls-secrets)
+
+```shell tab='Adapter Ingress'
+wso2.deployment.adapter.ingress.tlsSecretName=<TLS_CERT_SECRET_IN_THE_SAME_NAMESPACE>
+```
+
+```shell tab='Gateway Ingress'
+wso2.deployment.gatewayRuntime.router.ingress.tlsSecretName=<TLS_CERT_SECRET_IN_THE_SAME_NAMESPACE>
+```
+
+## Disable Test Token Issuer
+
+Choreo Connect provides the ability to generate a JWT as a test key to access the APIs. It is recommended to disable the `/testkey` endpoint in the Production environments.
+Refer the document [Generate a Test JWT]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/security/generate-a-test-jwt/) to read more about Test JWT.
+
+Set the following value to `false` when installing the helm chart or set it in the values.yaml file.
+
+```shell
+wso2.deployment.gatewayRuntime.enforcer.security.testTokenIssuer.enabled=false
+```
+
+## Create Custom Docker Image
```

Comment:
Make Adapter custom docker image

Reply:
Added separate sections for Standalone and Control Plane modes with https://github.com/wso2/docs-apim/pull/5214/commits/9519594eb8494b506a1b2318846d599edf1cf7f6